### PR TITLE
Add token to render the mandatory SEPA mandate info.

### PR DESCRIPTION
**Pull request by torotil**

### CODE REVIEW

File: stripe-payment-sepa-mandate-info.tpl.php [hunk 1]
```diff
@@ -13,5 +13,11 @@
  * - $iban_last4: Last 4 digits of the payee account’s IBAN.
  */
 ?>
-<h3>SEPA-Information</h3>
-<p>Die SEPA-Lastschrift erfolgt mit der Mandatsreferenz <?php echo $mandate_reference; ?> und der Creditor-ID <?php echo $creditor_id; ?> von deinem Konto mit den Endziffern ***<?php echo $iban_last4; ?>.</p>
+<h3><?php echo t('SEPA information'); ?></h3>
+<p><?php
+echo t('The SEPA direct debit is set up with the mandate reference @reference and the Creditor ID @creditor from your account with the final digits ***@last4.', [
```
Comment (katzmo) [inline stripe-payment-sepa-mandate-info.tpl.php:18]: I would find that sentence very confusing, maybe double-check with native speakers?

File: src/SepaController.php [hunk 2]
```diff
@@ -38,4 +41,28 @@ public function customerDataForm() {
...
+    return new SepaConfigurationForm();
+  }
+
+  /**
+   * Load the intent object and populate the $payment object accordingly.
+   */
+  protected function fetchIntent(\Payment $payment, Api $api) {
+    $intent = parent::fetchIntent($payment, $api);
+    $payment_method = $api->retrievePaymentMethod($intent['payment_method']);
+    $mandate = $api->retrieveMandate($intent['mandate']);
```
Comment (katzmo) [inline src/SepaController.php]: Those 3 consecutive API calls could be put into a singe call by expanding the SetupIntent object:
```
SetupIntent::retrieve([
  'id' => $id
  'expand' => ['payment_method', 'mandate'],
]);
```

**Review by katzmo — COMMENTED** [review]
Just out of curiosity, what made you decide to save the mandate data to the DB after all?

File: src/SepaController.php (hunk 2 above)
Comment (torotil) [reply src/SepaController.php]: Will look into this. Thanks!

File: stripe-payment-sepa-mandate-info.tpl.php (hunk 1 above)
Comment (torotil) [reply stripe-payment-sepa-mandate-info.tpl.php:18]: It is the text from the trello ticket, but I will double-check.

**Review by katzmo — APPROVED** [review]

**Comment by torotil** [comment]
> Just out of curiosity, what made you decide to save the mandate data to the DB after all?

Error-handling: This way there is one single place where errors from the API can happen. The situation is way more complex if an error can occur any time tokens are resolved.

**Pull request merged**